src/MusicalScore/Graphical/VexFlow/SvgVexFlowBackend.ts

```diff
@@ -190,6 +190,18 @@ export class SvgVexFlowBackend extends VexFlowBackend {
     }
 
     public renderCurve(points: PointF2D[]): Node {
+
+        //Here we want to make sure all points are numbers. If not, we will use the previous point.
+        //This avoids the "<path> attribute d: Expeced number." SVG error.
+        for (let i: number = 0; i < points.length; i++) {
```

Comment:
consider checking if `points.length > 1`, otherwise points[-1] is possible. And if `isNaN(points[0].x)`

Reply:
@alexshenker how do you get array.length === -1 on an array? I don't think I've ever seen that before.
The NaN check on .x is definitely a good idea.

Reply:
@alexshenker you mean if the array is undefined? I don't think that's possible within current OSMD code.
current osmd code is basically this:
```ts
const curvePointsInPixels: PointF2D[] = []; // in VexFlowMusicSheetDrawer.drawSlur()
//...
curvePoints.push(new PointF2D(...))
this.backend.renderCurve(curvePointsInPixels);
```
in above code, it's impossible for `curvePointsInPixels.length >= 0` to be false.
Maybe this can happen in your code?

Reply:
@sschmidTU Bad phrasing on my part.
I meant when calculating `points[i - 1].x`: if `i === 0` and if `points[0]` is`NaN`, line 198 would attempt to assign
`points[0].x = points[-1].x;`
I imagine this is improbable, but wouldn't hurt to handle this edge case.